frontends/web/src/locales/en/app.json

```diff
@@ -72,7 +72,7 @@
     "signing": "To proceed, confirm message on your BitBox02.",
     "success": {
       "message": "Proceed on {{host}}",
-      "title": "Successfully sent address to external service."
+      "title": "Message successfully confirmed and sent"
```

Comment:
:-1: again i don't think the "message" is important at all. why not `Address successfully sent`?